packages/next/src/server/stream-utils/node-web-streams-helper.ts

```diff
@@ -608,24 +641,35 @@ export async function continueDynamicPrerender(
       .pipeThrough(createStripDocumentClosingTagsTransform())
       // Insert generated tags to head
       .pipeThrough(createHeadInsertionTransformStream(getServerInsertedHTML))
+      // Insert generated metadata to body
+      .pipeThrough(createInsertedMetadataStream(getServerInsertedMetadata))
```

Comment:
If we are prerendering and the page is dynamic we want to omit the metadata altogether right? this way on resume we know that the metadata must be sent again consistently. If we sometimes include metadata in a dynamic prerender then we will also need to track whether it was sent for this particular prerender to decide to include/exclude it on the resume.